builder/scripts/mkimage-phan.bash

```diff
@@ -59,6 +59,7 @@ build() {
   } >&2
 
   # install php-ast
+  export CFLAGS=-O2
```

Comment:
Let's skip this for now, I'll look at doing some benchmarks.

Reply:
done